synapse/handlers/federation.py

```diff
@@ -182,7 +182,7 @@ def on_receive_pdu(self, origin, pdu, sent_to_us_directly=False):
         room_id = pdu.room_id
         event_id = pdu.event_id
 
-        logger.info("[%s %s] handling received PDU: %s", room_id, event_id, pdu)
```

Comment:
Why remove the prefixing? I find it quite useful to track how we're processing an event?

Reply:
because it's in the logcontext anyway.

Reply:
Oh, so it is. I had forgotten the log context isn't the generic request one. Carry on.